### DISCUSSION TITLE
HTML suggestions?

### DISCUSSION THREAD

**Discussion by hexisXz** [lapce/lapce#3232] [Q&A]
I was wondering if Lapce has HTML suggestions. I installed the HTML plugin but I am pretty sure it's just syntax highlighting. Oh and CSS.

**Comment by 2brownc** [comment]
Guess you need a language server for suggestions. 

* https://microsoft.github.io/language-server-protocol/implementors/servers/

**Comment by codebychandu** [comment]
If you’re looking for HTML suggestions, you might need to use a language server that supports HTML. You can explore language servers that implement the [Language Server Protocol]

https://microsoft.github.io/language-server-protocol/implementors/servers/

to get suggestions for HTML and other languages

 As for CSS, it’s possible that Lapce’s plugin provides syntax highlighting for CSS, but additional features like suggestions might require a separate language server or extension. Feel free to explore other plugins or extensions to enhance your HTML and CSS development experience! 😊🚀

**Comment by elo1lson** [comment]
Em relação ao CSS, existe um plugin, [lapce css](https://github.com/elo1lson/lapce-css)
